libs/designsystem/src/lib/components/icon/kirby-icon-registry.service.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { KirbyIconRegistryService } from './kirby-icon-registry.service';
+import { Icon, IconSettings } from './icon-settings';
+
+describe('KirbyIconRegistryService', () => {
+  let service: KirbyIconRegistryService;
+
+  beforeEach(() => {
+    service = new KirbyIconRegistryService();
+  });
+  it('should create service', () => {
+    expect(service).toBeTruthy();
+  });
+  describe('getCustomIcons', () => {
+    it('get icons from empty map', () => {
+      const expectedIcons: Icon[] = [];
+      expect(service.getCustomIcons()).toEqual(expectedIcons);
```

Comment:
```suggestion
      const emptyArray = [];
      expect(service.getCustomIcons()).toEqual(emptyArray);
```

Reply:
changed to const expectedIcons = [];
Dont like datatypes in varible name :-)

Reply:
Ok - but it's just not very clear that what you are REALLY expecting is that icons should be empty...
Something a long the lines of `expect(service.getCustomIcons()).toBeEmpty()` (if jasmine had that syntax)